packages/presentation/lib/src/app/widgets/side_nav_bar.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:presentation/generated/l10n.dart';
+import 'package:presentation/utils/images_container.dart';
+import 'package:presentation/utils/widget_display_helper.dart';
+
+class SideNavigationBar extends StatelessWidget {
+  const SideNavigationBar({
+    required this.selectedIndex,
+    required this.callback,
+    required this.isShowNavBar,
+    super.key,
+  });
+
+  final Function(int) callback;
+  final int selectedIndex;
+  final bool isShowNavBar;
+
+  @override
+  Widget build(BuildContext context) {
+    return !WidgetDisplayHelper.isBottomNavBarActive(context) && isShowNavBar
+        ? NavigationRail(
+            destinations: [
+              NavigationRailDestination(
+                icon: SvgPicture.asset(AssetsImages.movieReelIcon),
+                label: Text(S.of(context).navBarFilms),
+                selectedIcon: SvgPicture.asset(
+                  AssetsImages.movieReelIcon,
+                  color: Theme.of(context).colorScheme.onPrimaryContainer,
+                ),
+              ),
```

Comment:
optional - u can create the subclass for NavigationRailDestination, which will receive the icon path and label string as constructor params and map this data to the format which is needed for superclass constructor